internal/pkg/service/buffer/config/worker.go

```diff
@@ -16,6 +16,10 @@ const (
 	WorkerEnvPrefix = "BUFFER_WORKER_"
 	// DefaultCheckConditionsInterval defines how often it will be checked upload and import conditions.
 	DefaultCheckConditionsInterval = 5 * time.Second
+	// DefaultMinimalUploadInterval defines minimal interval between two slice upload operations.
+	DefaultMinimalUploadInterval = 5 * time.Second
+	// DefaultMinimalImportInterval defines minimal interval between two import file operations.
+	DefaultMinimalImportInterval = 30 * time.Second
```

Comment:
Podmienky sme povodne kontrolovali kazdych `30s`, lebo file import je storage job, a nechceli sme aby sa to tam frontovalo.

Preto som zaviedol tieto nove 2 nastavenia, aby sa dal samostatne nastavit minimalny interval medzi 2ma upload a import operaciami.